rocAL/include/decoders/audio/sndfile_decoder.h

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright (c) 2019 - 2024 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
@swetha097 Copyright must be only 2024 and not 2019 - 2024, Since we have introduced the file this year. Please make this change in all newly added files

Reply:
done for all files